tests/hw-accel/neuron/dra/tests/dra-migration-test.go

```
@@ -0,0 +1,461 @@
+
+					dsList, err := APIClient.K8sClient.AppsV1().DaemonSets(
+						params.NeuronNamespace).List(
+						context.TODO(), metav1.ListOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					dpFound := false
+
+					for _, ds := range dsList.Items {
+						if strings.HasPrefix(ds.Name, params.DevicePluginDaemonSetPrefix) {
+							dpFound = true
+							Expect(int(ds.Status.NumberReady)).To(BeNumerically(">", 0),
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (no shared variables above expr)

(wsl_v5)